G/GRASS/build_tarballs.jl

```diff
@@ -0,0 +1,534 @@
+    ExecutableProduct("d.erase", :d_erase, "grass78/bin"),
+    ExecutableProduct("d.labels", :d_labels, "grass78/bin"),
+    ExecutableProduct("d.histogram", :d_histogram, "grass78/bin"),
+    ExecutableProduct("d.colorlist", :d_colorlist, "grass78/bin"),
+    ExecutableProduct("d.his", :d_his, "grass78/bin"),
+    ExecutableProduct("d.title", :d_title, "grass78/bin"),
+    ExecutableProduct("d.profile", :d_profile, "grass78/bin"),
+    ExecutableProduct("d.grid", :d_grid, "grass78/bin"),
+    ExecutableProduct("d.rgb", :d_rgb, "grass78/bin"),
+    ExecutableProduct("d.graph", :d_graph, "grass78/bin"),
+    ExecutableProduct("d.legend", :d_legend, "grass78/bin"),
+    ExecutableProduct("d.rast", :d_rast, "grass78/bin"),
+    ExecutableProduct("d.colortable", :d_colortable, "grass78/bin"),
+
+    ExecutableProduct("ps.map", :ps_map, "grass78/bin"),
+
+    ExecutableProduct("test.r3flow", :test_r3flow, "grass78/bin"),
+    ExecutableProduct("test.raster3d.lib", :test_raster3d_lib, "grass78/bin"),
+
+    ExecutableProduct("echo", :echo, "grass78/etc"),
+    ExecutableProduct("run", :grass_run, "grass78/etc"),
+    ExecutableProduct("lock", :grass_lock, "grass78/etc"),
+    ExecutableProduct("current_time_s_ms", :current_time_s_ms, "grass78/etc"),
+    ExecutableProduct("clean_temp", :clean_temp, "grass78/etc"),
+
+    ExecutableProduct("m.cogo", :m_cogo, "grass78/bin"),
+    ExecutableProduct("m.transform", :m_transform, "grass78/bin"),
+    ExecutableProduct("m.measure", :m_measure, "grass78/bin"),
+    ExecutableProduct("m.nviz.script", :m_nviz_script, "grass78/bin"),
+
+    ExecutableProduct("t.connect", :t_connect, "grass78/bin"),
+
+    ExecutableProduct("dbf", :dbf, "grass78/driver/db"),
+    ExecutableProduct("sqlite", :sqlite, "grass78/driver/db"),
+    ExecutableProduct("ogr", :ogr, "grass78/driver/db"),
+    ExecutableProduct("seg", :seg, "grass78/etc/r.watershed"),
+    ExecutableProduct("ram", :ram, "grass78/etc/r.watershed"),
+    ExecutableProduct("vector", :vector, "grass78/etc/lister"),
+    ExecutableProduct("cell", :cell, "grass78/etc/lister")
+
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="GDAL_jll", uuid="a7073274-a066-55f0-b90d-d619367d196c"))
+    Dependency(PackageSpec(name="PROJ_jll", uuid="58948b4f-47e0-5654-a9ad-f609743f8632"))
+    Dependency(PackageSpec(name="Cairo_jll", uuid="83423d85-b0ee-5818-9007-b63ccbeb887a"))
+    Dependency(PackageSpec(name="FreeType2_jll", uuid="d7e528f0-a631-5988-bf34-fe36492bcfd7"))
+    Dependency(PackageSpec(name="LAPACK_jll", uuid="51474c39-65e3-53ba-86ba-03b1b862ec14"))
+    Dependency(PackageSpec(name="FFTW_jll", uuid="f5851436-0d7a-5f13-b9de-f02708fd171a"))
+    Dependency(PackageSpec(name="Libiconv_jll", uuid="94ce4f54-9a6c-5748-9c1c-f9c7231a4531"))
+    Dependency(PackageSpec(name="Xorg_xorgproto_jll", uuid="c4d99508-4286-5418-9131-c86396af500b"))
```

Comment:
This is a build dependency, these are only header files
```suggestion
    BuildDependency(PackageSpec(name="Xorg_xorgproto_jll", uuid="c4d99508-4286-5418-9131-c86396af500b"))
```